fix(add-space-overlay): Change design of add space modal

vikram-raj opened the pull request:
Story: https://openshift.io/openshiftio/Openshift_io/plan/detail/977

christianvogt commented on src/app/app.component.less:
```diff
@@ -1,3 +1,7 @@
 }
+
+.f8-modal-lg {
+  width: 70%;
```
See https://github.com/fabric8-ui/fabric8-ui/pull/3427#discussion_r237916289

christianvogt commented on src/app/app.component.html:
```diff
@@ -5,7 +5,14 @@
 </pfng-toast-notification-list>
-<f8-add-space-overlay class="launcher fade in" *ngIf="showAddSpaceOverlay"></f8-add-space-overlay>
+
+<div class="modal fade" bsModal #modalAddSpaceOverlay="bs-modal" tabindex="-1" role="dialog" aria-hidden="true">
```
See https://github.com/fabric8-ui/fabric8-ui/pull/3427#discussion_r237912501

christianvogt commented on src/app/space/add-space-overlay/add-space-overlay.component.html:
```diff
@@ -1,64 +1,96 @@
+              Learn more <i class="fa fa-external-link"></i>
+            </a>
+          </p>
+          <h5>What happens next?</h5>
```
Add a `<br/>` before this `<h5>`

christianvogt commented on src/app/space/add-space-overlay/add-space-overlay.component.less:
```diff
@@ -1,152 +1,36 @@
+  }
+  &--right-section {
+    border-left: 3px solid @color-pf-black-200;
+    padding-left: 40px;
```
Change to `padding: 0 40px;` to compensate for removing `col-sm-9`

christianvogt commented on src/app/space/add-space-overlay/add-space-overlay.component.html:
```diff
@@ -1,64 +1,96 @@
-            <button id="cancelSpaceButton" class="btn btn-link" aria-label="Cancel"
-                    (click)="hideAddSpaceOverlay()">Cancel</button>
+      <section class="col-sm-6 f8-add-space-overlay--right-section">
+        <div class="col-sm-9">
```
`col-sm-9` adds too much dead space on the right. We shouldn't constrain the text at all here. Remove.

christianvogt commented on src/app/space/add-space-overlay/add-space-overlay.component.html:
```diff
@@ -1,64 +1,96 @@
+            and participate in creating the product.
+          </p>
+        </div>
+        <div class="font-size-small col-sm-9">
```
`col-sm-9` adds too much dead space on the right. We shouldn't constrain the text at all here. Remove.

You also need some vertical spacing between this text block and the one above.

christianvogt commented on src/app/space/add-space-overlay/add-space-overlay.component.less:
```diff
@@ -1,152 +1,36 @@
+  &-label {
+    font-weight: normal;
+  }
+  &--container {
```
Elements should use double underscore and not double dash.

christianvogt commented:
Please remove all the font-size-small. We should be using the standard font-size.
Except for the error message under the name input field.

vikram-raj commented:
@christianvogt I addressed all the comments in https://github.com/fabric8-ui/fabric8-ui/pull/3421/commits/dfa123a3a3391112b50e8c9b4203acc1d6c18d96. Please review it again.

joshuawilson commented:
@fabric8-ui/docs Please review. This will impact the documentation.

fabric8cd commented:
:tada: This PR is included in version 3.0.0 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/fabric8-ui)
- [GitHub release](https://github.com/fabric8-ui/fabric8-ui/releases/tag/v3.0.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
